egs2/audioset/codec1/conf/tuning/train_dac.yaml

```diff
@@ -0,0 +1,168 @@
+##########################################################
```

Comment:
For start, we may simply keep the configuration that we used for the recipe (aka, remove unused ones)